AudioKit/Common/Nodes/Playback/Player/ClipPlayer/AKClip.swift

```diff
@@ -41,9 +41,10 @@ extension AKClip {
     }
 }
 
-/// A file based AKClip
+/// A file based AKClip, with a completion handler.
 @objc public protocol FileClip: AKClip {
     var audioFile: AKAudioFile { get }
+    var completionCallback: AKCallback? { get }
```

Comment:
Since this is already an objc protocol, I think it would be better to make this addition optional.
```
@objc optional 
var completion: AKCallback? { get }
```